src/main/resources/config/application-test.properties

```diff
@@ -22,4 +22,10 @@ spring.thymeleaf.view-names= *
 spring.thymeleaf.suffix=.html
 spring.thymeleaf.mode=HTML5
 spring.thymeleaf.encoding=UTF-8
-spring.thymeleaf.content-type=text/html
\ No newline at end of file
+spring.thymeleaf.content-type=text/html
+
+##AREA FIFTYLAN SETTINGS
+
+a5l.paymentReturnUrl=https://areafiftylan.nl/ordersuccess
```

Comment:
This is meant as a front-end URL. You can't redirect a user to the api ;). The front-end would make a call to the API.